src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -1,90 +1,68 @@
 <template>
-<multipane class="c-imagery-layout"
-           type="vertical"
->
-    <pane :style="{'min-height': `300px`}">
-        <div class="main-image-wrapper c-imagery has-local-controls">
-            <div class="h-local-controls h-local-controls--overlay-content c-local-controls--show-on-hover l-flex-row c-imagery__lc">
-                <span class="holder flex-elem grows c-imagery__lc__sliders">
-                    <input v-model="filters.brightness"
-                           class="icon-brightness"
-                           type="range"
-                           min="0"
-                           max="500"
-                    >
-                    <input v-model="filters.contrast"
-                           class="icon-contrast"
-                           type="range"
-                           min="0"
-                           max="500"
-                    >
-                </span>
-                <span class="holder flex-elem t-reset-btn-holder c-imagery__lc__reset-btn">
-                    <a class="s-icon-button icon-reset t-btn-reset"
-                       @click="filters={brightness: 100, contrast: 100}"
-                    ></a>
-                </span>
-            </div>
-
-            <div class="main-image s-image-main"
-                 :class="{'paused unnsynced': paused(),'stale':false }"
-                 :style="{'background-image': `url(${getImageUrl()})`,
-                          'filter': `brightness(${filters.brightness}%) contrast(${filters.contrast}%)`}"
-            >
-            </div>
-
-            <div class="l-image-controller flex-elem l-flex-row">
-                <div class="l-datetime-w flex-elem grows">
-                    <a class="c-button show-thumbs sm hidden icon-thumbs-strip"></a>
-                    <span class="l-time">{{ getTime() }}</span>
-                </div>
-                <div class="h-local-controls flex-elem">
-                    <a class="c-button icon-pause pause-play"
-                       :class="{'is-paused': paused()}"
-                       @click="paused(!paused())"
-                    ></a>
-                </div>
+<div class="c-imagery">
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.

Reply:
See previous comment.